helm/hpcc/templates/_helpers.tpl

```diff
@@ -2200,3 +2200,94 @@ Fills "result" dictionary with "planeCategories" and "namedPlanes"
 {{- $_ := set .result "planeCategories" $planeCategories -}}
 {{- $_ := set .result "namedPlanes" $namedPlanes -}}
 {{- end -}}
+
+{{/*
+A template to generate a HorizontalPodAutoscaler for a named  workload resource (such as a Deployment or StatefulSet)
+Pass in dict with workload resource name and kind (Deployment|ReplicaSet|StatefulSet|ReplicationController), and hpa section
+*/}}
+{{- define "hpcc.addHPABehavior" }}
+{{- if .behavior -}}
+  {{- if hasKey .behavior "stabilizationWindowSeconds" }}
```

Comment:
minor: there may be exceptions already, but we have said we will use 1 space indentation for templating (vs yaml) in our helm chart templates.